wolfssl/include.am

```diff
@@ -16,10 +16,16 @@ nobase_include_HEADERS+= \
                          wolfssl/certs_test.h \
                          wolfssl/test.h \
                          wolfssl/version.h \
-                         wolfssl/options.h \
                          wolfssl/ocsp.h \
                          wolfssl/crl.h
 
 noinst_HEADERS+= \
                          wolfssl/internal.h
 
+# For distro build don't install options.h.
```

Comment:
The Debian package is already eliminating the options.h from the build because it causes issues with the multi-arch cross-compile feature.

Here is Felix's feedback on this when I talked with him about it on 11/22:

"It prevents Multi-Arch from working. Multi-Arch is a cross-compile feature that allows your clients to compile ARM binaries on any Debian PC.

There are at least two solutions:

1. Move to /usr/include/<multiarch-tuple>/ (please see Multi-Arch Tuple Specification https://wiki.debian.org/Multiarch/Tuples)
Advantage: Natural location for an architecture-dependent header.
Disadvantage: File is never used when compiling

2. Move to $datadir (or another appropriate place, see autoconf Directory Variables https://www.gnu.org/prep/standards/html_node/Directory-Variables.html)
Advantage: Great location for an architecture-dependent data file
Disadvantage: May be hard to find"

@toddouska, let me know your thoughts for these solutions and I can adjust the code accordingly for the distro build.